### PR TITLE
Update transfer-data-set.feature

### PR DESCRIPTION
**Pull request by joel-simpson**
reviewed with Jesus and Peter and update language for clarity. still work in progress.

### CODE REVIEW

**Comment by joel-simpson** [comment]
With latest updates the Gherkin now reflects real behaviour much more closely. However in reviewing the code I noticed a couple of things not reflected here... not sure if these are material enough to include in the Gherkin or not.  a) is timeout settings for the API call and b) is logic around how to name the temporary directory for a dataset if one using that name already exists (solution is to append with a number, up to 20). While these will add detail to the Gherkin, I'm inclined to say we should include them because they are important error scenarios and we should be testing that they work as expected.

File: features/Jisc/transfer-data-set.feature [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
-#	d) recreating original folder hierarchy... PeterVG commented that required info may come in future version of MetadataCreate message?	 
+Scenario: Successful DataSet Transfer
+  Given archivematica_channel_adaptor has received a valid Preservation Request
+	# as described by register-preservation-request.feature	
   
-# The following two lines are from Peter's data mapping analysis. Can these wait to "Ingest Metadata" in sprint 4?
-# RDSS-Archivematica creates a new archivematica:intellectualEntity metadata record and assigns it a unique archivematica:identifier.
-# RDSS-Archivematica reads the property values stored in the rdss:dataset metadata record and writes them a new archivematica:intellectualEntity record using the mappings specified below.
+  When archivematica_channel_adaptor creates a temporary directory inside the standardTransfer directory
+    # in the archivematica_transfer_deposit_dir (as per root.go)
```
Comment (jhsimpson) [inline features/Jisc/transfer-data-set.feature:11]: not sure I understand this reference 'root.go'

**Review by jhsimpson — APPROVED** [review]
other than the single comment/question I posted, this looks like an accurate reflection of MVP workflow.

File: features/Jisc/transfer-data-set.feature (hunk 1 above)
Comment (joel-simpson) [reply features/Jisc/transfer-data-set.feature:11]: This is just a reference to where archivematica_transfer_deposit_dir / standardTransfer directory is described in the code. Probably not needed, but I will leave for now as we will be doing many more iterations on this so lots of opportunity to tweak and improve.

**Pull request merged**